test/AutoDiff/validation-test/differentiable_protocol_requirements.swift

```diff
@@ -1,6 +1,9 @@
 // RUN: %target-run-simple-swift
 // REQUIRES: executable_test
 
+// Disabled due to test failure with `-O`: SR-13250.
+// REQUIRES: SR13250
```

Comment:
How about `XFAIL: swift_test_mode_optimize`?

Reply:
Thanks, let's try that!